docs/how_to_contribute/math/how_to_add_prime_fields.md

```diff
@@ -17,6 +17,10 @@ Choose the [prime field generator](/tachyon/math/finite_fields/generator/prime_f
    - `namespace`: The selected namespace, commonly set as `tachyon::math`.
    - `class_name`: The name of the class, usually the same as the name of the prime field (in PascalCase).
    - `modulus`: The modulus value of the prime field.
+   - `reduce32`: If the prime field uses a special reduction function, this property holds the string version of the code which overrides the default 32-bit reduction function.
+   - `reduce64`: If the prime field uses a special reduction function, this property holds the string version of the code which overrides the default 64-bit reduction function.
+   - `use_asm`: If `ffiasm` fails to generate the assembly code, setting this flag to false is recommended.
```

Comment:
I meant to keep the first sentence as well: "This flag indicates whether assembly is used."